styles.css

```diff
@@ -93,7 +98,253 @@ ul.navbar-items {
   line-height: 1.5rem;
 }
 
-@media only screen and (max-width: 375px) {
+/* Works section styling */
+
+#works-section {
+  margin: 10px 0;
+}
+
+.box-container {
+  display: flex;
+  margin: 50px 200px 50px 200px;
+}
+
+.left-section {
+  flex: 1;
+}
+
+.left-section img {
+  float: left;
+  margin-right: 30px;
+  max-width: 100%;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 10px;
+  margin: 0 200px;
+}
+
+.grid-item {
+  position: relative;
+  background-color: #eaeaea;
+  height: 400px;
+}
```

Comment:
- [ ] Excellent applying styles to your work cards 🚀. However, please kindly ensure to use the same `font-family` and `font-color` suggested in the Figma design 🤓. Also, please fix the margins and paddings. 🔧

Figma design 👇🏽
![f1](https://github.com/Xghayor/Portfolio/assets/116701485/7a1b6924-f670-4831-b305-8977a9f5ab1f)


Your design 👇🏽
![d1](https://github.com/Xghayor/Portfolio/assets/116701485/bcd72985-e4c5-4b1f-9eb9-b66f7644d467)